recipes/h5glance/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "h5glance" %}
+{% set org = "European-XFEL" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: dd848a967112f0a0d1d04a1739afbe7dc7bac682b93b8017e5d5a11ca8260c5d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - h5glance = {{ name }}.terminal:main
+    - h5glance-html = {{ name }}.html_cli:main
+
+requirements:
+  host:
+    - python >=3.7
+    - flit >=3.2,<4
```

```suggestion
    - flit-core >=3.2,<4
```

This is the build backend piece of Flit.